private/buf/buflsp/completion.go

```diff
@@ -462,14 +354,46 @@ func completionItemsForField(ctx context.Context, file *file, declPath []ast.Dec
 				),
 			)
 		}
-		if len(iters) == 0 {
-			return nil
+	case ast.DefKindService:
+		if showKeywords {
+			iters = append(iters,
+				keywordToCompletionItem(
+					serviceLevelKeywords(),
+					protocol.CompletionItemKindKeyword,
+					tokenSpan,
+					offset,
+				),
+			)
+		}
+	case ast.DefKindMethod:
+		if showKeywords {
+			iters = append(iters,
+				keywordToCompletionItem(
+					optionKeyword(),
+					protocol.CompletionItemKindKeyword,
+					tokenSpan,
+					offset,
+				),
```

Comment:
~~should we also show `stream` here? (and message types?)~~

Ah, nvm, played with it: this is for within the brackets, where `option` is:

```proto
  rpc Download(DownloadRequest) returns (DownloadResponse) {
    option idempotency_level = NO_SIDE_EFFECTS;
  }
```